ui/pb/helpers.go

```diff
@@ -110,14 +110,13 @@ func GetFixedLengthDuration(d, maxDuration time.Duration) (result string) {
 	return string(buf[i:])
 }
 
-// Clampf returns the given value, "clamped" to the range [min, max].
-// This is copied from lib/util.go to avoid circular imports.
-func Clampf(val, min, max float64) float64 {
+// Clampf returns the given value, "clamped" to the range [floor, ceiling].
+func Clampf(val, floor, ceiling float64) float64 {
```

Comment:
Just as an observation, it does sound like something we might unexport. We don't use it outside of the `pb` package.

Reply:
I expect we will do this in one big sweep one day ... soon :tm: ;)